### DISCUSSION TITLE
Pandoc support?

### DISCUSSION THREAD

**Discussion by xahodo** [vkbo/novelWriter#1205] [Ideas]
This would be helpful in converting the document to a million (well, ok, a bit exaggerated) formats (among which Word). Most editors, agents, and publishers expect Word documents these days. It would be a shortcut to supporting Word as an output format.

Maybe somebody would prefer LaTeX output (for whatever reason)? That's in the palette of tastes too.

**Comment by vkbo** [comment]
Pandoc support has been discussed before, and it will likely not happen. The main conclusion from last time this was discusses is that Pandoc can already convert from several of the formats novelWriter outputs, so the possibility is already there.

As for Word documents, the support is already there in terms of the Open Document standard which all the major word processors can read. If you need to convert further into a proprietary format, then all you have to do is pass the document through one of those word processors. These complex document formats are a lot of work to add and support. I prefer to focus on open formats that have well defined specs.

**Reply by vkbo** [reply]
By the way, Pandoc has been tested with novelWriter in the past. There was an integration with it which was dropped in version 0.6. It was a pretty basic integration, but it was dropped primarily because it was a very difficult tool to work with and it produced very bad results in terms of layout and formatting.

**Comment by johnblommers** [comment]
Dropping direct Pandoc support is IMHO the right move.

novelWriter's focus should IMHO remain on enabling the writer to write and write well.

Therefore novelWriter should do as little about formatting as possible.

Formatting is a whole 'nother phase and it can be complicated. That's why Pandoc is difficult to use. It's a helluva Swiss Army Knife that's also very good at translating file formats in the presence of templates. It's also a commandline tool. That alone would chase away a large number of writers.

And that is why we have Vellum on  the Macintosh.

**Reply by vkbo** [reply]
Yeah, I remember there were several issues with the Pandoc integration. I remember discussing these with you up until the new build tool was added and Pandoc dropped.
